CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 -   Open task drawer directly in cp details page (#122)
 -   Algo creation events aren't included in newsfeed anymore (#127)
 -   Renamed any tuple thing into a task thing (#129)
+-   Update compute plan failed task key
```

Comment:
Could you include the PR number here?

Reply:
Done https://github.com/Substra/substra-frontend/pull/134/commits/23ad981906579103d0a1c8150179dca0fef3c82c